Tests/PointFreeTests/AccountTests/__Snapshots__/UpdateProfileTests/testUpdateEmailSettings.email_settings_after_update.txt

```diff
@@ -2,4 +2,5 @@
   ▿ EmailSetting
     - newsletter: Newsletter.newEpisode
     ▿ userId: 00000000-0000-0000-0000-000000000001
-      - rawValue: 00000000-0000-0000-0000-000000000001
+      ▿ rawValue: 00000000-0000-0000-0000-000000000001
+        - uuid: "00000000-0000-0000-0000-000000000001"
```

Comment:
I'm not sure why these snapshots changed when I re-recorded everything. They are also the only failures on this branch. 🤔